tools/run_tests/performance/loadtest_config.py

```diff
@@ -217,6 +218,15 @@ def expect_stream_start(self):
                 self.write_indent()
                 self.write_indicator(header_comment, need_whitespace=False)
 
+        def expect_block_sequence(self):
```

Comment:
do we really need custom formatting for the outputs? what does that buy us?
Seems like all we need is to generate valid .yaml and this custom formatting is just more code to maintain.

same below.

Reply:
The source for the templates is going to be written by hand somewhere. Right now we are adding to grpc/test-infra templates to use with prebuilt images for the various languages. These may need to be understood, compared and modified in the future. These files in grpc/test-infra are used to generate "all-language" templates. It is useful that the format of the source templates is consistent among themselves, and also with the generated templates and the final configurations. This is useful for a human reading the source files, or for tools calculating diffs on the generated files and for testing. I intend to make the format of all files consistent. This change is consistent with common tools (e.g. "prettier"), so it will reduce the maintenance burden if compared with having a format that would always change when you apply formatting tools, or tools that might generate output that is scrambled in relation to their input.